backend/routes/auth.js

```diff
@@ -158,6 +160,15 @@ router.post('/resendverification', async (req, res) => {
   }
 });
 
+// Delete unverified users every minute
+schedule.scheduleJob('*/1 * * * *', async () => {
+  await User.destroy({
+    where: {
+      emailVerified: false,
+      createdAt: { [Op.lt]: new Date(Date.now() - 60000) }
```

Comment:
can we actually do like 10 minutes? Or will that be too hard to test? Now that we know it works don't think 1 minutes is sufficient for real users

Reply:
Yes, I have changed the time to 10 minutes.